types/src/version.rs

```diff
@@ -42,30 +47,42 @@ const MAINNET_SCHEDULE: [Upgrade; 5] = [
         height: UPGRADE_TAPE_HEIGHT,
         network: NetworkVersion::V5,
     },
+    Upgrade {
+        height: UPGRADE_KUMQUAT_HEIGHT,
+        network: NetworkVersion::V6,
+    },
 ];
 
 /// Specifies the network version
-#[derive(Debug, PartialEq, Clone, Copy, PartialOrd)]
+#[derive(Debug, PartialEq, Clone, Copy, PartialOrd, Serialize_repr)]
 #[repr(u32)]
 pub enum NetworkVersion {
     /// genesis (specs-actors v0.9.3)
-    V0,
+    V0 = 0,
     /// breeze (specs-actors v0.9.7)
-    V1,
+    V1 = 1,
     /// smoke (specs-actors v0.9.8)
-    V2,
+    V2 = 2,
     /// ignition (specs-actors v0.9.11)
-    V3,
+    V3 = 3,
     /// actors v2 (specs-actors v2.0.x)
-    V4,
+    V4 = 4,
     /// tape (increases max prove commit size by 10x)
-    V5,
-    /// kumquat (specs-actors v2.2.0)
-    V6,
+    V5 = 5,
+    // kumquat   (specs-actors v2.2.0)
+    V6 = 6,
```

Comment:
none of the integers matter, but if it is to be added, should also add to V7 (these values are already implied though)